### DISCUSSION TITLE
How to dock query results tab at the bottom

### DISCUSSION THREAD

**Discussion by urko-b** [mtxr/vscode-sqltools#1209] [Q&A]
![image](https://github.com/mtxr/vscode-sqltools/assets/11888892/80acf715-0bca-4278-8cae-f08010cd1dcd)

Hi I want to move this results window on the bottom where the terminal window is. How can I do this? 

Blessings.

**Comment by gjsjohnmurray** [comment]
It's not possible to move that to the Panel, but you can drag it and drop it onto the lower half of the query tab to place it below rather than to the right.

**Reply by urko-b** [reply]
ok, it would be great to have this behavior. Do you think it could be possible in a future release?

**Comment by gjsjohnmurray** [comment]
It would be a lot of work to support Results appearing in the Panel rather than in the editor area.

The request at #1163 to make it configurable to appear below rather than to the right should be relatively easy to do. I suggest you subscribe to that issue.
